docs/secure-connections/sauce-connect-5/quickstart.md

```diff
@@ -15,30 +15,71 @@ The Sauce Connect Proxy version 5 major release introduces CLI changes. Please r
 An [`sc legacy`](/dev/cli/sauce-connect-5/legacy/) command is introduced to help users transition from version 4.x.x to 5.0.x.
 :::
 
+:::note
+[Service accounts](/basics/acct-team-mgmt/managing-service-accounts) are not supported by Sauce Connect Proxy and cannot start or manage tunnels. Only user accounts can manage tunnels.
+:::
+
 ## What You'll Need
 
 - A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
 - If you haven't already, make sure you can access the website or mobile app that you'll be testing from the Sauce Connect Proxy host.
 - Check to see if you have any [proxies](/secure-connections/sauce-connect-5/guides/proxies/) that are required to access the public Internet.
 
-## Installing Sauce Connect Proxy
+## Installation and Configuration
+
+- See the [installation instructions](/secure-connections/sauce-connect-5/installation/)
+- See the [configuration guide](/secure-connections/sauce-connect-5/guides/configuration/)
+
+### Run Sauce Connect Proxy
+
+Every Sauce Connect Proxy tunnel spins up a fresh virtual machine (VM) that is used only for your tests. Once the tunnel is closed, VMs are destroyed.
+
+1. Make sure that the directory containing the `sc` binary (or `sc.exe`, for Windows) is in the `$PATH`. Otherwise, you will have to specify the path to the binary in the command line, i.e. `/path/to/sc`
```

Comment:
for windows it's `sauce-connect.exe` by default. Users can set alias to `sc.exe` but I doubt everyone will know that.